optimade/validator/validator.py

```diff
@@ -643,7 +665,11 @@ def _construct_single_property_filters(
                 reversed_response, _ = self._get_endpoint(
                     f"{endp}?filter={reversed_query}",
                     multistage=True,
+                    optional=query_optional,
                 )
+                if not reversed_response:
```

Comment:
The same is true for this response as my previous comment.

Reply:
:point_up: